1.10/monitoring/debugging/slow-docker.md

```diff
@@ -0,0 +1,42 @@
+---
+post_title: Slow Docker Apps and Deployments
+menu_order: 40
+---
+
+If you recently upgraded to DC/OS 1.10 or configured `MESOS_CGROUPS_ENABLE_CFS=true` in your Mesos agent configuration, you may see slow-running Docker applications or slow deployments.
+
+# Strict CPU limitations are now default
+
+When using Apache Mesos, you can choose to use either CPU shares or strict CPU limitations. If you use CPU shares, your task can consume more CPU cycles than initially configured in your Marathon app definition, if your host system has free CPU cycles. If you are using strict CPU limitations, your task can only consume a maximum of CPU time based on your Marathon configuration.
+
+The default Mesos configuration uses CPU shares. CFS (Completely Fair Scheduler) strict CPU limitation was introduced as a default in DC/OS a while ago, but until recently this configuration is only respected by the Mesos executor and not by the Docker executor. [MESOS-6134](https://issues.apache.org/jira/browse/MESOS-6134) was fixed in the latest Mesos release, which is also included in DC/OS 1.10.
```

Comment:
I'm used to introducing an acronym after the term:  Completely Fair Scheduler (CFS)

I suggest removing the "a while ago", as I don't think it adds anything.

Suggestion for the last sentence:

Completely Fair Scheduler (CFS) strict CPU limitations is the default in DC/OS, but until recently this configuration was respected only by the Mesos executor and not by the Docker executor. The fix [MESOS-6134](https://issues.apache.org/jira/browse/MESOS-6134) in the latest Mesos release and also included in DC/OS 1.10 removes this limitation.

Reply:
👍